src/main/java/net/minestom/server/entity/LivingEntity.java

```diff
@@ -387,12 +387,12 @@ public boolean damage(@NotNull DamageType type, float value) {
             // play damage sound
             final Sound sound = type.getSound(this);
             if (sound != null) {
-                SoundCategory soundCategory;
+                Source soundCategory;
                 if (this instanceof Player) {
-                    soundCategory = SoundCategory.PLAYERS;
+                    soundCategory = Source.PLAYER;
                 } else {
                     // TODO: separate living entity categories
-                    soundCategory = SoundCategory.HOSTILE;
+                    soundCategory = Source.HOSTILE;
                 }
 
```

Comment:
Why are you replacing the Minestom SoundCategory with Source?
Because this is linked to the MC protocol, I believe Minestom should keep its own enum.
Adding a `toAdventureSource()` to the Sound enum could help porting.

Reply:
Yes this is a fair point! I suppose the reason why I swapped these is because Adventure's `Source` is an exact replica of Minestom's `SoundCategory` enum. I didn't think there was much point having them both, especially considering that `Source` is used to play sounds, etc. Happy to swap back to `SoundCategory` if you feel it's best.